### ISSUE TITLE
Reveal slideshows are clipped on the right hand side in the browser

### ISSUE THREAD

**Issue by dov** [ipython/ipython#8629]
When converting an ipynb to a reveal slide show, the result shown in the browser is offset to the right hand side to such an extent that part of the contents is clipped. The Ascii illustration shows the rendering in the browser window. 

Note that there is a second problem that may or may not be related, that first and second slides are not shown as separate slides, but on the same page. But perhaps that is related to my assumption that a slide is started by a markdown top level (one #) headline.

        +----------------------------------------+
        |                                        |
        |                                        |
        |                                        |
        |                            First slide |
        |                            This is an i|
        |                                        |
        |                    In [1]: import panda|
        |                            print pd.Dat|
        |                                        |
        |                               A  B     |
        |                            0  1  2     |
        |                            1  3  4     |
        |                                        |
        |                            A second sli|
        |                            * Some bulle|
        +----------------------------------------+

The slides were created with the command line:

    ipython nbconvert myslides.ipynb  --to slides --post serve --ServePostProcessor.reveal_cdn="http://cdn.jsdelivr.net/reveal.js/2.5.0"

**Comment by damianavila** [comment]
Can you post/link your notebook as well... thanks

**Comment by dov** [comment]
After writing the above, I checked my assumption that slides are mapped to top level head lines, and I found that I was wrong, and that there is a separate slide attribute that has to be set. Once I set the slide attribute everything worked as expected! 

The problem was that I probably related to the fact that I was interacting with the ipython server through emacs-ipython-notebook, which I realize causes various problems with the resulting ipynb file.

In any case I can live with the idea of postprocessing the notebook in a webbrowser and then setting the resulting slide page attributes, so I'm closing this bug.
